x/liquidationsV2/types/codec.go

```diff
@@ -39,6 +44,7 @@ var (
 func init() {
 	RegisterLegacyAminoCodec(amino)
```

Comment:
RegisterCodec or RegisterLegacyAminoCodec ? This should be common across module

Reply:
done